lisa/runner.py

```diff
@@ -257,77 +287,69 @@ def _start_loop(self) -> None:
         except StopIteration:
             self._log.debug(f"no more runner found, total {len(remaining_runners)}")
 
-        if self._runners:
-            run_message = notifier.TestRunMessage(
-                status=notifier.TestRunStatus.RUNNING,
-            )
-            notifier.notify(run_message)
+        if not self._runners:
```

Comment:
This line is not necessary now. We can remove both this check and the for loop in Ln 285. What we need to do is to add `has_more_runner` into the while conditions. So it will loop at least once. The code will be simpler.